### ISSUE TITLE
Open source plan?

### ISSUE THREAD

**Issue by cuijh26** [Guohanzhong/OSA-LCM#1]
Great work! Do you have a plan to release the code? thanks.

**Comment by EYcab** [comment]
+1

**Comment by Guohanzhong** [comment]
I will release the inference code soon and pulish a latest version paper

**Comment by animemory** [comment]
> I will release the inference code soon and pulish a latest version paper

update here?

**Comment by johndpope** [comment]
Excited to see this -
You maybe interested in this  bytedance paper recreation 
https://github.com/johndpope/OmniHuman-1-hack/tree/main/seaweed_apt

**Comment by Guohanzhong** [comment]
Interesting codebase from @johndpope ,
We recently release a new work, outperform the OminiHuman, distillated with DMD2, 
https://arxiv.org/pdf/2503.05978

**Comment by Guohanzhong** [comment]
> > I will release the inference code soon and pulish a latest version paper
> 
> update here?

Yes, will release the inference codebase within this month, and the latest version will support real-time chatting.

**Comment by alinaSarwar** [comment]
Any update on the release plan?

**Comment by michaellee1** [comment]
+1!

**Comment by Pudge-tao** [comment]
+1

**Comment by animemory** [comment]
> > > I will release the inference code soon and pulish a latest version paper
> > 
> > 
> > update here?
> 
> Yes, will release the inference codebase within this month, and the latest version will support real-time chatting.

Hi, Any update on release time about the inference and training
